### ISSUE TITLE
Empty cells when custom table created with <Thead> and <Th> components

### ISSUE THREAD

**Issue by straku** [glittershark/reactable#237]
When I use custom table by passing `columns` property to `Table` component everything seems to be working as expected. Here's array passed to `columns` property:
```
const columns = [
  {key: 'name', label: 'Name'},
  {key: 'type', label: 'Media type'},
  {key: 'date', label: 'Date created'},
  {key: 'createdBy', label: 'Created by'},
  {key: 'action', label: 'Action'}
]
```
And `Table` rendering:
```
<Table columns={columns}>
  {data.map((item, i) => {
    return <Tr key={i}>
      <Td column="name">{item.name}</Td>
      <Td column="type">{item.type}</Td>
      <Td column="date" value={item.date}>{parseDate(item.date)}</Td>
      <Td column="createdBy">{item.createdBy}</Td>
      <Td column="action"><ActionDelete/></Td>
    </Tr>
  })}
</Table>
```
But when I try to change it and use `Thead` and `Th` components to specify custom header for the table the only thing that table is rendering is header and empty cells (right number of rows and cells but without any content). Here's example of the table:
```
<Table sortable={sortable}>
  <Thead>
    <Th column="name">Name</Th>
    <Th column="type">Media type</Th>
    <Th column="date">Date created</Th>
    <Th column="createdBy">Created by</Th>
    <Th column="action">Action</Th>
  </Thead>
  {data.map((item, i) => {
    return <Tr key={i}>
      <Td column="name">{item.name}</Td>
      <Td column="type">{item.type}</Td>
      <Td column="date" value={item.date}>{parseDate(item.date)}</Td>
      <Td column="createdBy">{item.createdBy}</Td>
      <Td column="action"><ActionDelete/></Td>
    </Tr>
  })}
</Table>
```
Based on the documentation I believe that this is the right way of creating custom table. Can you please point me to the direction what am I doing wrong in above example?

**Comment by jflay** [comment]
I am also getting this issue... tried using hard-coded data and result is the same.

**Comment by cklab** [comment]
The PR should address this issue but as a workaround, if you specify an element (any will do) for your children instead of just a string, it should render:
```
<Table sortable={sortable}>
  <Thead>
    <Th column="name"><span>Name</span></Th>
    <Th column="type"><span>Media type</span></Th>
    <Th column="date"><span>Date created</span></Th>
    <Th column="createdBy"><span>Created by</span></Th>
    <Th column="action"><span>Action</span></Th>
  </Thead>
  {data.map((item, i) => {
    return <Tr key={i}>
      <Td column="name">{item.name}</Td>
      <Td column="type">{item.type}</Td>
      <Td column="date" value={item.date}>{parseDate(item.date)}</Td>
      <Td column="createdBy">{item.createdBy}</Td>
      <Td column="action"><ActionDelete/></Td>
    </Tr>
  })}
</Table>
```

**Comment by glittershark** [comment]
Closed in #242
